plugins/UsersManager/tests/Integration/UsersManagerTest.php

```diff
@@ -76,6 +76,7 @@ private function _checkUserHasNotChanged($user, $newPassword, $newEmail = null,
         }
         $userAfter = $this->api->getUser($user["login"]);
         unset($userAfter['date_registered']);
+        unset($userAfter['ts_password_modified']);
```

Comment:
Does it make sense to hide this from general API output? I have a feeling like it better should be but not 100% sure or can't explain it. @mattab @diosmosis  , eg in `UsersManager\API::enrichUser`?

Reply:
It might be useful to show the user later (eg, your password hasn’t been updated in N days!). Doesn’t HAVE to be in API output for this to work though.

Reply:
Would actually leave the `ts_password_modified` in the API output as some people may find value in this @diosmosis

Reply:
Maybe only for super users then? I am personally finding this information quite private and wouldn't want an admin to see that kind of information.

Reply:
Will display it in API output if the user requested is the current user or the super user. Let me know if there are any issues w/ this.

Reply:
Actually that seems to be the current case: https://github.com/piwik/piwik/blob/3.x-dev/plugins/UsersManager/API.php#L362 ? @mattab @tsteur am I wrong? lf there's something to change here, let me know.

Reply:
what needs changed is probably remove this line `unset($userAfter['ts_password_modified']);` from the test?

Reply:
I think that value is non-deterministic and would cause test failures if it's not removed during tests.

Reply:
maybe just check the value is there and don't test the actual value?

Reply:
ok, i'll add that check, and I won't make any modifications to UsersManager/API since it already restricts user info to either the superuser or the currently logged in user.